templates/include/mozdata/base.1.schema.json

```diff
@@ -0,0 +1,3 @@
+"apiVersion": {
```

Comment:
Is API version the only thing we want to record here (I think we had this discussion in the MozData PR as well). I could imagine: Spark version, Scala version, Metastore URL. That may be it off the top of my head.

Reply:
I suppose? It seems odd to me that we have to get data review for internal tools.